onnxruntime/core/providers/openvino/openvino_execution_provider.cc

```diff
@@ -198,4 +198,13 @@ std::vector<AllocatorPtr> OpenVINOExecutionProvider::CreatePreferredAllocators()
 }
 #endif
 
+common::Status OpenVINOExecutionProvider::SetEpDynamicOptions(gsl::span<const char* const> keys,
+                                                              gsl::span<const char* const> values) {
+  if ((std::find(keys.begin(), keys.end(), "ep.dynamic.workload_type") != keys.end()) &&
```

Comment:
Also needs implementation for Setting Dynamic Option -> ep.dynamic.workload_type to "DEFAULT"

Reply:
Added the changes